dvd/dvd_dump.ixx

```diff
@@ -417,15 +419,25 @@ export bool redumper_dump_dvd(Context &ctx, const Options &options, DumpMode dum
                 auto layer_descriptor = (READ_DVD_STRUCTURE_LayerDescriptor &)structure[sizeof(CMD_ParameterListHeader)];
 
                 physical_sectors_count += get_layer_length(layer_descriptor);
-                pfi_data_start = sign_extend<24>(endian_swap(layer_descriptor.data_start_sector));
             }
 
             // Kreon drives return incorrect sectors count
             if(physical_sectors_count != sectors_count)
             {
-                LOG("warning: READ_CAPACITY / PHYSICAL sectors count mismatch, using PHYSICAL");
-                is_xbox = ctx.drive_config.vendor_specific.starts_with("KREON V1.00");
                 sectors_count = physical_sectors_count;
+
+                if(ctx.drive_config.vendor_specific.starts_with("KREON V1.00"))
+                {
+                    status = cmd_kreon_set_lock_state(*ctx.sptd, KREON_LockState::LOCKED);
```

Comment:
Looking back on it, I am not sure if this is actually effective as we are doing this after having read the PFI